pcdsdevices/epics/slits.py

```diff
@@ -44,13 +66,26 @@ class Slits(IocDevice):
     close_cmd = Component(EpicsSignal, ":CLOSE")
     block_cmd = Component(EpicsSignal, ":BLOCK")
 
+    #Nominal
+    nominal_aperature = 5.0
```

Comment:
Any thoughts on making this a parameter? Some of the scientists open the slits to 10mm. I'm not sure it really matters.

Reply:
👍